examples/utils/bundle-size-tests/bannedModulesPlugin.js

```diff
@@ -0,0 +1,64 @@
+/*!
```

Comment:
Does this file need to be here? I.e. should it be separate PR?

Reply:
It keeps it from regressing, I can move it to another PR if that helps review this one.